templates/js/copilot-plugin-from-scratch-api-key/README.md

```diff
@@ -0,0 +1,59 @@
+# Overview of Custom Search Results app template
+
+## Build a message extension from a new API with Azure Functions
+
```

Comment:
"Build a message extension from a new API with Azure Function and API key authentication"

Reply:
Updated